plugin/plugins.vim

```diff
@@ -30,7 +30,7 @@ def RegisterPlugins()
     Register('abbrev', ['*'], 10)
     Register('buffer', ['*'], 11)
     Register('path', ['*'], 12) # higher priority than buffer, so /xx/yy work
-    Register('dictionary', ['text', 'markdown'], 5)
```

Comment:
What happens when you open a python file and there is no dictionary associated with it? I think default should be very conservative. Users can set these options in .vimrc by using `filetypes` option.
```
dictionary: { enable: true, priority: 10, filetypes: ['*'] }
```